h2o-py/tests/testdir_algos/glrm/pyunit_PUBDEV_7609_GLRM_loading_name.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.glrm import H2OGeneralizedLowRankEstimator
+
+
+def glrm_arrests():
+  print("Importing USArrests.csv data...")
+  arrestsH2O = h2o.upload_file(pyunit_utils.locate("smalldata/pca_test/USArrests.csv"))
+  arrestsH2O.describe()
+  loading_name = "are_we_confused_yet"
+  
+  print("H2O initial Y matrix:\n")
+  initial_y = [[5.412,  65.24,  -7.54, -0.032],
+               [2.212,  92.24, -17.54, 23.268],
+               [0.312, 123.24,  14.46,  9.768],
+               [1.012,  19.24, -15.54, -1.732]]
+  initial_y_h2o = h2o.H2OFrame(list(zip(*initial_y)))
+  initial_y_h2o.show()
+
+  print("H2O GLRM on de-meaned data with quadratic loss:\n")
+  glrm_h2o = H2OGeneralizedLowRankEstimator(k=4, transform="DEMEAN", loss="Quadratic", gamma_x=0, gamma_y=0, 
+                                            init="User", user_y=initial_y_h2o, recover_svd=True,
+                                            loading_name=loading_name)
+  glrm_h2o.train(x=arrestsH2O.names, training_frame=arrestsH2O)
+  assert loading_name == str(glrm_h2o._model_json["output"]['representation_name']), \
```

Comment:
I think this line demonstrates the issue of this approach, there is an inconsistency between the input/output API

since loading_frame was never used, I would suggest this:
1. expose representation_name
2. map loading_name to representation name in the V3 class
3. deprecate loading_name and modify the documentation to use representation_name
4. temporarily keep both loading_name and representation_name
5. delete loading_name from the API

Reply:
I like this approach.  Let me put it in.